java/maven/src/org/netbeans/modules/maven/TestChecker.java

```diff
@@ -59,6 +59,17 @@
                     config.setProperty("test", test + '#' + method);
                 }
         }
+        if (ActionProviderImpl.COMMAND_INTEGRATION_TEST_SINGLE.equals(action) ||
+            ActionProviderImpl.COMMAND_DEBUG_INTEGRATION_TEST_SINGLE.equals(action) ||
+            "profile-tests".equals(action)) 
+        { //NOI18N - profile-tests is not really nice but well.
```

Comment:
Please move the NOI18N onto the line where the string is located (one line up). Else it is not connected to that string. (Yes I know that there other locations in that file, that are missing NOI18N markers or have them on the next line, but ...)

Reply:
@abourree I saw the updated commit - would you mind squashing the to commits into one?